packages/windows/src/RPA/Windows/__init__.py

```diff
@@ -485,7 +485,7 @@ def __init__(self):
         libraries = [
             ActionKeywords(self),
             ElementKeywords(self),
-            LocatorKeywords(self),
+            LocatorKeywords(self, kwargs.get("locators_path", None)),
```

Comment:
This is a matter of taste, but I'd almost have all of the library attributes stored in this container class, and accessing them through `self.ctx `in the keyword classes.

Reply:
Take a look at this pending PR for instance: https://github.com/robocorp/rpaframework/pull/373

Reply:
That's because you're accessing the locators from multiple classes, right? Here they're required in the `LocatorKeywords` only, but like you said, it's about a taste of opening them more to a global access or restricting to the place they're used only. (until further access extension required)

Reply:
True enough. Actually the only thing that's a bit of a downside in that diff is the use of **kwargs, which prevents IDEs and documentation from showing what the actual library init arguments are. But that isn't related to where they are stored.